src/benchmarklib/synthetic_table_generator.cpp

```diff
@@ -74,30 +72,33 @@ std::shared_ptr<Table> SyntheticTableGenerator::generate_table(
       static_cast<size_t>(std::ceil(static_cast<double>(num_rows) / static_cast<double>(chunk_size)));
 
   // add column definitions and initialize each value vector
-  TableColumnDefinitions column_definitions;
+  auto column_definitions = TableColumnDefinitions{};
   for (auto column_id = size_t{0}; column_id < num_columns; ++column_id) {
+    // NOLINTBEGIN(bugprone-unchecked-optional-access)
+    // False positive, we literally check that the optional holds a value in the same line.
```

Comment:
Far too many false positives for my taste. Temporarily disable the rule?

Reply:
Could live with that.